packages/union-component/src/components/DonationPaymentForm.tsx

```diff
@@ -44,13 +47,31 @@ const DonationPaymentForm: React.FC<Props> = ({
   const [paymentProvider, setPaymentProvider] = useState<
     DonationPaymentProvider | undefined
   >();
+  const [phoneNumberError, setPhoneNumberError] = useState('');
   const errorMessage: string | undefined = errors?.join(' ');
 
+  const isPhoneNumberValid = (phoneNumber: string) => {
+    const formattedNumber = phoneNumber.split(' ').join('');
```

Comment:
Why we do this split/join, are we trying to do like `phoneNumber.replaceAll(' ', '')` kinda thing?

Reply:
No specific reason, it's just another way to achieve the same result. I'd like more `replaceAll`